src/frontend/html/create-themes.md

```diff
@@ -111,6 +111,48 @@ How to deploy your files depends on your host application:
 Laravel and Symfony
 : Run `composer up`
 
+To avoid having to use `composer update` or `composer up` when changing the CSS and JS (development environment) files in the theme extension, a symbolic link can be made with the following command:
+```bash
+ln -s /packages/<extname>/themes/client/html/<extname> /public/vendor/shop/themes/<extname>
+```
+
+With this, the changes you make to your package will be published automatically.
+
+Also, you can implement "hot reload" with Laravel Mix and Browser Sync:
+
+```bash
+npm install laravel-mix --save-dev
+```
+```bash
+npm install -g browser-sync
+```
+
+webpack.mix.js:
+```javascript
+mix.browserSync({
+    injectChanges: true,
+    proxy: 'example.com',
+    hostname: 'example.com',
+    port: 3000,
+    ignored: /node_modules/,
+    files: [
+        'config/shop.php',
+        'packages/**/config/*.php',
+        'public/**/*.css',
+        'resources/**/*',
+        'lang/*.json',
+        'packages/**/templates/admin/jqadm/**/*.php',
+        'packages/**/views/*.php',
+        'packages/**/templates/client/html/**/**/*.php',
+        'packages/**/templates/client/html/**/**/**/*.php',
+        'packages/**/themes/client/html/**/*.css',
+        'packages/**/themes/client/html/**/*.js',
+        'packages/**/themes/admin/jqadm/*.css',
+        'packages/**/themes/admin/jqadm/*.js'
```

Comment:
Just a question: Would be `*` for the package name enough too because it's only one directory level.
Also, as `**` should match all sub-directories, is it possible to shorten the config to:
```
        'config/shop.php',
        'lang/*.json',
        'packages/**/*',
        'public/assets/*',
        'public/vendor/**/*',
        'resources/**/*',
```

Reply:
Yes! It could be as: 

```
        'public/**/*.css',
        'resources/**/*',
        'lang/*.json',
        'packages/**/*.{php,css,js}'
```